.env.docker

```diff
@@ -0,0 +1,4 @@
+DATABASE_HOST=db
+DATABASE_PORT=5432
+DATABASE_USERNAME=postgres
+DATABASE_URL=postgresql://postgres@db:5432
```

Comment:
If anyone can think of a more elegant way to do this, I'm all ears.. 
`wait-for-it.sh` needs the hostname and port, but the application needs the `DATABASE_URL` to be set. I tried interpolating it in the `docker_web` file but then it was only set on the application thread, and rake files or scripts run with `docker exec` didn't have it set.